### PR TITLE
feat: enable showing relations in Graph view for files

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by erdkse** [comment]
https://user-images.githubusercontent.com/2957624/230819472-b56161fa-0551-4495-9f15-2b410b93d938.mov

**Comment by topliceanurazvan** [comment]
From the video seems like when trying to select a file it doesn't select it and the resource remains selected, looks like a bug to me...

**Comment by topliceanurazvan** [comment]
Also I saw that clicking on one file showed the `source` editor tab empty, is that a bug?

**Comment by erdkse** [comment]
> From the video seems like when trying to select a file it doesn't select it and the resource remains selected, looks like a bug to me...

Nope, I didn't see any problem when trying to click, just the navigator pane doesn't scroll to the resource.

**Comment by erdkse** [comment]
> Also I saw that clicking on one file showed the `source` editor tab empty, is that a bug?

It doesn't select the source tab, yes it is a bug but I am planning to fix it in a separate PR.

File: src/components/molecules/ResourceGraphTab/ResourceGraphTab.tsx [hunk 1]
```diff
@@ -19,33 +20,46 @@ const ResourceGraphTab: React.FC = () => {
   const dispatch = useAppDispatch();
   const activeStorage = useAppSelector(activeResourceStorageSelector);
   const selectedResource = useSelectedResource();
-  const resources = useMemo(() => (selectedResource ? [selectedResource] : []), [selectedResource]);
   const activeResoureMetaMap = useActiveResourceMetaMap();
   const activeResoureContentMap = useActiveResourceContentMap();
+  const selection = useAppSelector(state => state.main.selection);
```
Comment (topliceanurazvan) [inline src/components/molecules/ResourceGraphTab/ResourceGraphTab.tsx:25]: Lets use here `useSelectorWithRef` and use `selectionRef` for `onSelectResource` and `onSelectImage`

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**